spec/components/provider_interface/application_card_component_spec.rb

```diff
@@ -151,6 +152,38 @@
     end
   end
 
+  describe 'relative_date_text_color' do
```

Comment:
A system test to check for a CSS colour? Wouldn't a spec for the `relative_date_text_color` method in `task_view_helper_spec` be enough?

Reply:
Yeah, I can move the spec to the helper. I added this spec before I added the Helper spec.